cop4610/makefile

```diff
@@ -0,0 +1,2 @@
+p1: main.c
+	gcc main.c -std=c11 -o p1
```

Comment:
cc1: error: unrecognized command line option "-std=c11"
    make: *** [p1] Error 1

This still doesn't work. It's so much easier to just not support c99 for this project.

That means:

    for(int i = 0; ;) { }

becomes 

    int i;
    for(i = 0 ; ;) { }